public/app/features/trails/ActionTabs/LayoutSwitcher.tsx

```diff
@@ -14,23 +16,25 @@ export type LayoutType = 'single' | 'grid' | 'rows';
 
 export class LayoutSwitcher extends SceneObjectBase<LayoutSwitcherState> {
   public Selector({ model }: { model: LayoutSwitcher }) {
-    const { active, options } = model.useState();
+    const { options } = model.useState();
+    const { layout } = sceneGraph.getAncestor(model, MetricScene).useState();
 
     return (
       <Field label="View">
-        <RadioButtonGroup options={options} value={active} onChange={model.onLayoutChange} />
+        <RadioButtonGroup options={options} value={layout} onChange={model.onLayoutChange} />
       </Field>
     );
   }
 
   public onLayoutChange = (active: LayoutType) => {
-    this.setState({ active });
+    locationService.partial({ layout: active });
```

Comment:
Can we let url sync system do this ? We need to support data trails in the embedded (drawer) mode where url sync is disabled